ArchiSteamFarm/WebBrowser.cs

```diff
@@ -136,14 +137,18 @@ public sealed class WebBrowser : IDisposable {
 					break;
 				}
 
-				if (string.IsNullOrEmpty(response.Content)) {
+				if (response.Content == null) {
 					continue;
 				}
 
 				T obj;
 
 				try {
-					obj = JsonConvert.DeserializeObject<T>(response.Content);
+					using StreamReader sr = new StreamReader(response.Content);
```

Comment:
Nitpicking, but I don't like abbreviations that do not mean anything, even if it's obvious from the code what it is. I'd rename it to `streamReader` and below to `jsonReader`.